src/context/IIIFResourceContext.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { ReactNode, createContext, useContext, useEffect, useMemo, useState } from 'react';
-import { getNavResourceItemId, isFirstResourceItem, isLastResourceItem } from '../lib/helpers';
+import { getResourceItemIndex, isFirstResourceItem, isLastResourceItem } from '../lib/helpers';
```

Comment:
I like the rename of this helper.

Reply:
So it's not really a rename, but a different helper function we're importing to get the `index` value instead of the `id` value.  But we're using `getResourceItemIndex` to get the `id` value.